src/app/portlets/content-types/fields/content-types-fields-list/content-types-fields-list.component.ts

```diff
@@ -30,12 +29,25 @@ export class ContentTypesFieldsListComponent implements OnInit {
                 take(1)
             )
             .subscribe((fields: FieldType[]) => {
-                this.fieldTypes = fields.map((fieldType: FieldType) => {
+                const LIVE_DIVIDER_CLAZZ =
+                    'com.dotcms.contenttype.model.field.ImmutableLineDividerField';
+
+                const mappedFields = fields.map((fieldType: FieldType) => {
                     return {
                         clazz: fieldType.clazz,
                         name: fieldType.label
                     };
                 });
+
+                const fieldsFiltered = mappedFields.filter(
+                    (field) => field.clazz !== LIVE_DIVIDER_CLAZZ
+                );
+
+                const LINE_DIVIDER = mappedFields.find(
+                    (field) => field.clazz === LIVE_DIVIDER_CLAZZ
+                );
+
+                this.fieldTypes = [COLUMN_BREAK_FIELD, LINE_DIVIDER, ...fieldsFiltered];
```

Comment:
Why LINE_DIVIDER is filter first and then is add again?

Reply:
Because @fishsmith said so 😀